### PR TITLE
NAS-114525 / 22.12 / NAS-114525: Fixing memory leaks in enabledWhile/disabledWhile.

### PR DESCRIPTION
**Pull request by undsoft**
Turns out `enabledWhile/disabledWhile` creates a subscription that you need to unsubscribe yourself.
Code review should be enough.

### CODE REVIEW

File: src/app/pages/system/advanced/syslog-form/syslog-form.component.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ import { advancedConfigUpdated } from 'app/store/system-config/system-config.act
 })
 export class SyslogFormComponent implements OnInit {
   isFormLoading = false;
+  subscriptions: Subscription[] = [];
```
Comment (RehanY147) [inline src/app/pages/system/advanced/syslog-form/syslog-form.component.ts:27]: No `UntilDestroy({ arrayName: 'subscription' })`?

**Review by RehanY147 — CHANGES_REQUESTED** [review]

File: src/app/pages/system/advanced/syslog-form/syslog-form.component.ts (hunk 1 above)
Comment (undsoft) [reply src/app/pages/system/advanced/syslog-form/syslog-form.component.ts:27]: Good catch. Fixed.

**Review by RehanY147 — APPROVED** [review]
Looks good

**Review by AlexKarpov98 — APPROVED** [review]
LGTM! 👍

**Pull request merged**